tests/noisers/test_atom_types_noiser.py

```diff
@@ -0,0 +1,77 @@
+import einops
+import pytest
+import torch
+
+from diffusion_for_multi_scale_molecular_dynamics.noisers.atom_types_noiser import (
+    AtomTypesNoiser,
+)
+
+
+@pytest.mark.parametrize("shape", [(10, 1), (4, 5, 3), (2, 2, 2, 2)])
+class TestNoisyAtomTypesSampler:
+
+    @pytest.fixture(scope="class", autouse=True)
+    def set_random_seed(self):
+        torch.manual_seed(23423)
+
+    @pytest.fixture()
+    def num_atom_types(self):
```

Comment:
call this `num_classes` (which is `num_atom_types+1`). Otherwise, the names collide with what the functions expect.

Of course it makes no difference for the code, but it will help the reader understand the code if we use consistent and meaningful names for the variables.